index.html

```diff
@@ -18,19 +18,14 @@
 			<img src = "img/Kurken.jpg" id = "currentMap" />
 		</div>
 	</div>
-	<div class = "selection">
-		<div class = "languageDropdown">
+	<div class = "selection">		
```

Comment:
Remove the extra tabs at the end

Reply:
I have updated the branch and moved the locale selector at the top. But I don't see this extra tab.